docs/changelog-fragments.d/801.deprecation.rst

```diff
@@ -0,0 +1,8 @@
+Deprecated :py:meth:`~cheroot.ssl.Adapter.bind` method from the :py:class:`~cheroot.ssl.Adapter`
+interface and respective implementations. Previously :py:meth:`~cheroot.ssl.Adapter.bind` was
+doing nothing in the `:py:class:`builtin TLS adapter <cheroot.ssl.builtin.BuiltinSSLAdapter>`
+but was being used in the :py:class:`~cheroot.ssl.pyOpenSSLAdapter` to wrap the socket.
```

Comment:
Looks like I made some typos making suggestiosn from browser.
```suggestion
but was being used in the :py:class:`~cheroot.ssl.pyopenssl.pyOpenSSLAdapter` to wrap the socket.
```

*(looks like this is the only problem in Sphinx build: https://app.readthedocs.org/projects/cheroot/builds/30475668/#295926113--60)*